internal/manifests/collector/container.go

```diff
@@ -104,6 +104,15 @@ func Container(cfg config.Config, logger logr.Logger, otelcol v1alpha1.OpenTelem
 		},
 	})
 
+	if len(otelcol.Spec.ConfigMaps) > 0 {
+		for keyCfgMap := range otelcol.Spec.ConfigMaps {
+			volumeMounts = append(volumeMounts, corev1.VolumeMount{
+				Name:      naming.ConfigMapExtra() + otelcol.Spec.ConfigMaps[keyCfgMap].Name,
```

Comment:
`naming.ConfigMapExtra() + otelcol.Spec.ConfigMaps[keyCfgMap].Name` is used both here and in the volume spec, so it should be a separate function imo. Maybe add a parameter to `naming.ConfigMapExtra` and calculate it there?

Reply:
Following your suggestion, I've created a parameter to validate the name and return it for both places.